versioneer.py

```diff
@@ -1748,9 +1747,9 @@ def do_setup():
     ipy = os.path.join(os.path.dirname(cfg.versionfile_source), "__init__.py")
     if os.path.exists(ipy):
         try:
-            with open(ipy, "r") as f:
+            with open(ipy) as f:
                 old = f.read()
-        except EnvironmentError:
+        except OSError:
```

Comment:
Change of error from EnvironmentError to OSError. According to the [Built-in Exceptions docs](https://docs.python.org/3/library/exceptions.html#OSError.filename2), EnvironmentError was merged into OSError in 3.3